src/mono/mono/mini/interp/transform.c

```diff
@@ -4417,6 +4417,43 @@ interp_method_compute_offsets (TransformData *td, InterpMethod *imethod, MonoMet
 	td->il_locals_size = offset - td->il_locals_offset;
 	td->total_locals_size = offset;
 
+#ifdef MONO_ARCH_HAVE_SWIFTCALL
+	// Allocate SwiftError
+	if (mono_method_signature_has_ext_callconv (sig, MONO_EXT_CALLCONV_SWIFTCALL)) {
+		imethod->swift_error_offset = -1;
+
+		MonoClass *swift_error = mono_class_try_get_swift_error_class ();
+		MonoClass *swift_error_ptr = mono_class_create_ptr (m_class_get_this_arg (swift_error));
+
+		int swift_error_index = -1;
+		for (int i = 0; i < sig->param_count; i++) {
```

Comment:
Is it possible to avoid another loop over params and do this in the loop before?